osu.Game.Rulesets.Catch.Tests/TestSceneCatchPlayerLegacySkin.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.IO.Stores;
+using osu.Game.Skinning;
+using osu.Game.Tests.Visual;
+
+namespace osu.Game.Rulesets.Catch.Tests
+{
+    [TestFixture]
+    public class TestSceneCatchPlayerLegacySkin : PlayerTestScene
```

Comment:
Seeing that you've gone through the trouble of adding a legacy skin `PlayerTestScene`, mind making it more abstract (`LegacySkinPlayerTestScene`) so that it can be re-used for other rulesets?